runtime/rococo/src/lib.rs

```diff
@@ -418,7 +417,7 @@ impl pallet_session::Config for Runtime {
 	type ValidatorIdOf = ValidatorIdOf;
 	type ShouldEndSession = Babe;
 	type NextSessionRotation = Babe;
-	type SessionManager = pallet_session::historical::NoteHistoricalRoot<Self, ProposeParachain>;
+	type SessionManager = ();
```

Comment:
I think this is right, but can I get a second confirmation?

I don't see that `SessionManager` is implemented anywhere else.

Reply:
How will we add new validators?

Reply:
ah i must have misunderstood you.

I thought you said all validators are parity now, so i thought they might just be in genesis?

What function do we need here? Just a set validators function?

Reply:
`ProposeParachain` also had some sudo functions for adding/removing validators manually. We have and will continue to use that to scale up Rococo over time.

Reply:
I've updated `PrposeParachain` into `ValidatorManager`, which has only this functionality. I wanted to strip away any of the special management of Parachains so they are not accidentally used.

Let me know if this is all okay now, or if you think we should keep more features of propose Parachains.